client/state/signup/steps/website-content/test/reducer.ts

```diff
@@ -0,0 +1,408 @@
+import { expect } from 'chai';
+import { SIGNUP_COMPLETE_RESET } from 'calypso/state/action-types';
+import {
```

Comment:
The file rename is broken on git hub :/ 
So I broke the file rename into a seperate commit so the changes are visible clearly.

Check c37454185a4fa1eaf7be1b1e0e6be36996883e3f for the changes done to the test files.